src/map/src/items/inventory.cpp

```diff
@@ -125,6 +125,7 @@ ReturnValue Items::add_item(EntitySystem& entitySystem, RoseCommon::Entity entit
                 return ReturnValue::NO_SPACE;
             }
         }
+        entitySystem.delete_entity(item);
```

Comment:
Not here, we shouldn't delete the entity if we just added it as a separate item. We need to delete it only if we add it to the stack of other items

Reply:
Fixed 👍